### PR TITLE
Fix: Nacos client logDir path seperator

### PR DESCRIPTION
**Pull request by AlbumenJ**
**What this PR does**:

- fix logDir path seperator in Windows
- fix unit test prepare script for Windows

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:
no

### CODE REVIEW

File: config_center/nacos/client.go [hunk 1]
```diff
@@ -50,6 +51,15 @@ type NacosClient struct {
...
 
+func init() {
+	switch runtime.GOOS {
+	case "windows":
+		logDir = "logs\\nacos\\log"
+	default:
+		logDir = "logs/nacos/log"
+	}
+}
+
```
Comment (zouyx) [inline config_center/nacos/client.go:62]: Inaddition to set default value for `logDir`, we should provide a config item to change this value

Comment (AlbumenJ) [reply config_center/nacos/client.go:63]: done

File: config_center/nacos/client.go [hunk 2]
```diff
@@ -50,6 +51,16 @@ type NacosClient struct {
...
 }
 
+// init default Nacos Log dir, it can be override when create client
+func init() {
+	switch runtime.GOOS {
+	case "windows":
+		logDir = "logs\\nacos\\log"
+	default:
+		logDir = "logs/nacos/log"
+	}
```
Comment (Patrick0308) [inline config_center/nacos/client.go:61]:
```suggestion
	logDir = filepath.Join("logs", "nacos", "log")
```

Comment (AlbumenJ) [reply config_center/nacos/client.go]: thx, I have changed it

**Pull request merged**